manifests/service.pp

```diff
@@ -49,4 +49,66 @@
                   ],
   }
 
+  # if the Sentry config changes, do a full restart of the Sentry Celery workers
+  exec { 'restart-sentry-worker':
+    command     => '/usr/bin/systemctl stop sentry-worker; /usr/bin/systemctl start sentry-worker',
+    path        => '/bin:/usr/bin',
+    refreshonly => true,
+    subscribe   => Class['::sentry::config'],
+  }
+
+  # Sentry Celery Beat
+  file { '/etc/systemd/system/sentry-beat.service':
+    ensure  => present,
+    mode    => '0644',
+    owner   => 'root',
+    group   => 'root',
+    content => template('sentry/sentry-beat.service.erb'),
+    notify  => Exec['enable-sentry-services'],
+  }
+
+  service { 'sentry-beat':
```

Comment:
At one point the documentation mentioned the `-B` option which is why we've been using it :)  I suppose I prefer the current worker configuration as it keeps the puppet module a bit less complex.